README.md

```diff
@@ -144,12 +144,12 @@ In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning
 To make it easier to reference these feature flags in code, we recommend to define feature flag variables like below.

```

Comment:
> we recommend to define feature flag variables like below

I'm thinking, is it worth a recommendation? This is fairly basic definition of strings. I would imagine just removing this section would be fine.

The only issue I can think with that is that we reference `MyFeatureFlags.FeatureX` and others below. Perhaps though, it's clear enough that those references should be the names of the flags.

Reply:
If we want to remove the recommendation, maybe we can just use string "FeatureX" instead of the reference.
If I am a user, I can understand that this is just the name of the feature flag which is a string variable. 
But there is a customer who raised an issue for this. He expected us to update the recommendation.